web/portal/brand/src/entities/RetailAccount/ForeignKeyGetter.tsx

```diff
@@ -1,6 +1,8 @@
 import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
 import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
 import { RetailAccountPropertyList } from './RetailAccountProperties';
+import { ResidentialSelectOptions } from '../Company/SelectOptions';
```

Comment:
We need a RetailAccountSelectOptions here, otherwise only Residential clients are listed.